make.sh

```diff
@@ -28,23 +36,46 @@ kill_port_80_usage () {
     fi
 }
 
+find_hostfile () {
+    # try windows first, mac second; add future platforms here.
+    # make sure there aren't any conflicts e.g. the windows bash host may also provide an /etc/hosts file
+    index = 0
+    for hosts_candidate in "C:/windows/system32/drivers/etc/hosts" "/etc/hosts"; do
+        echo -n "looking for hostfile at "$hosts_candidate
+        if [ -f "$hosts_candidate" ]; then
+            #file exists, assign hostfile in outer function to current value
+            echo -n "Found host file at ${hosts_candidate}" 
+            
+            hostfile=$hosts_candidate
+            [[ "$index" == 0 ]] && $platform = APPLE
+            [[ "$index" == 1 ]] && $platform = WINDOWS
+
+            return;
+        fi
+        index = 1 + $index
+    done
+
+    echo "host file not found!"
+    exit 404
```

Comment:
```suggestion
    exit 1
```
With no special meaning, or usage, just use 1 as the exit code to indicate an error. Error codes greater than an 8-bit integer (255) can cause unwanted behavior.